src/Keboola/StorageApi/TableExporter.php

```diff
@@ -168,20 +169,27 @@ public function exportTables(array $tables = array())
                 throw new Exception('Missing destination');
             }
             if (!isset($table['exportOptions'])) {
-                $table['exportOptions'] = array();
+                $table['exportOptions'] = [];
             }
             if (empty($table['exportOptions']['columns'])) {
                 $tableDetail = $this->client->getTable($table['tableId']);
                 $table['exportOptions']['columns'] = $tableDetail['columns'];
             }
+            $exportOptions[$table['tableId']] = $table['exportOptions'];
 
+            $table['exportOptions']['gizp'] = true;
             $jobId = $this->client->queueTableExport($table['tableId'], $table['exportOptions']);
             $exportJobs[$jobId] = $table;
         }
         $jobResults = $this->client->handleAsyncTasks(array_keys($exportJobs));
         foreach ($jobResults as $jobResult) {
             $exportJob = $exportJobs[$jobResult['id']];
-            $this->handleExportedFile($exportJob['tableId'], $jobResult['results']['file']['id'], $exportJob['destination'], $exportJob['exportOptions']);
+            $this->handleExportedFile(
```

Comment:
Pokud bysme forcovali kompresi, tak pak by se musí upravit/zjednodušit `handleExportedFile`. Je tam teď pár ifů kvůli té kompresi.